records_mover/records/hints.py

```diff
@@ -52,6 +52,8 @@ def cant_handle_hint(fail_if_cant_handle_hint: bool, hint_name: str, hints: Reco
     # But let's be ready if they change their minds:
     'UTF-8': 'UTF8',
     'Windows-1252': 'CP1252',
+    # even if the only data it saw was in ASCII, let's be ready to see more
+    'ascii': 'UTF8',
```

Comment:
can't recall-- any case things to worry about here?

Reply:
("case" as in upper/lower)

Reply:
Nah, this is the correct case on both sides - it made the error message go away (chardet returns "ascii"), and Python spells UTF-8 as "UTF-8".